1.1/spec/18-Annex-C.adoc

```diff
@@ -68,3 +211,8 @@ The following table maps the functions and properties from Simple Features for S
 | Centroid(): Point | http://www.opengis.net/def/function/geosparql/centroid[geof:centroid] | 1.1 | http://www.opengis.net/ont/geosparql#hasCentroid[geo:hasCentroid] | 1.1
 | PointOnSurface(): Point | N/A | - | N/A | -
 |===
+
+
+==== References
```

Comment:
Can you perhaps just place this Reference in the main [Bibliography](https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/spec/19-Bibliography.adoc)? We don't have separate References sections for other sections.

Reply:
Yes, I will change it accordingly